nautobot/ui/src/App.js

```diff
@@ -60,9 +61,11 @@ function App() {
     return (
         <NautobotUIProvider theme={theme}>
             <BrowserRouter>
-                <Layout>
-                    <NautobotRouter />
-                </Layout>
+                <FiltersPanelContextProvider>
+                    <Layout>
+                        <NautobotRouter />
+                    </Layout>
+                </FiltersPanelContextProvider>
```

Comment:
It feels inelegant to me that we are adding functionality that should be exclusive to the object-list view at this basic level of the UI. Is there any cleaner way to do this?

Reply:
Although it is exclusive to the object list view, filters panel is at the same time an upper-level layout element, which is actually higher in the node hierarchy than the object list view itself. It would be either required to restructure the root layout of the application, or filters panel, to make the two (object list view and filters panel) coupled more tightly. I'm not insisting on this implementation though, just pointing out that we would need to change some of the requirements in order to do so

Reply:
Would it make logical to provide a placeholder (GlobalState, CreateContext, etc.) so that each page can override it as needed? 
The buttons that should be placed in the placeholder may then be provided by ObjectListView.